imod/mf6/wel.py

```diff
@@ -429,19 +434,21 @@ def to_mf6_pkg(
 
         wells_df = self._create_wells_df()
         nwells_df = len(wells_df["id"].unique())
-        wells_assigned = self._assign_wells_to_layers(wells_df, active, top, bottom, k)
-
-        nwells_assigned = (
-            0 if wells_assigned.empty else len(wells_assigned["id"].unique())
-        )
-
         if nwells_df == 0:
-            raise ValueError("No wells were assigned in package. None were present.")
+            raise ValidationError(
+                "No wells were assigned in package. None were present."
+            )
 
-        if not is_partitioned and nwells_df != nwells_assigned:
-            raise ValueError(
-                "One or more well(s) are completely invalid due to minimum conductivity and thickness constraints."
+        wells_assigned = self._assign_wells_to_layers(wells_df, active, top, bottom, k)
+        filtered_ids_in_assign = self.gather_filtered_well_ids(wells_assigned, wells_df)
```

Comment:
I would rename the above variables to:
`assigned_wells`
`filtered_assigned_wells` or `filtered_wells_ids`
I think that reads a bit easier

Reply:
Done